discovery-infra/test_infra/helper_classes/kube_helpers/global_vars.py

```diff
@@ -1,13 +0,0 @@
-from tests.conftest import env_variables
-
-# TODO - Temporary import - for backward compatibility
-from test_infra.consts.kube_api import *
-
-# TODO - remove all env variables dependency
-DEFAULT_API_VIP = env_variables.get('api_vip', '')
-DEFAULT_API_VIP_DNS_NAME = env_variables.get('api_vip_dns_name', '')
-DEFAULT_INGRESS_VIP = env_variables.get('ingress_vip', '')
-
-DEFAULT_MACHINE_CIDR = env_variables.get('machine_cidr', '')
-DEFAULT_CLUSTER_CIDR = env_variables.get('cluster_cidr', '172.30.0.0/16')
```

Comment:
what about all of those? They aren't used?

Reply:
@lalon4 can you confirm that?

Reply:
@YuviGold I checked their repository

Reply:
we dont use those envs, thanks for the update